corehq/apps/reports/standard/inspect.py

```diff
@@ -143,6 +143,13 @@ def display_in_dropdown(cls, domain=None, project=None, user=None):
         else:
             return True
 
+    @property
+    def template_context(self):
```

Comment:
What would you think of defining this in `ProjectInspectionReport`? Then you wouldn't need to do it twice, and I think conceptually it makes sense for all inspect-type reports to pass along user type names.

Reply:
Hm, do you mean in `ProjectReport`? It seems like `ProjectInspectionReport` isn't subclassed by these

Reply:
Added to `ProjectReport`: https://github.com/dimagi/commcare-hq/pull/22079/commits/74a7933d911eab3375d99f171ab5b49f9218c4e9